core/tests/puppeteer-acceptance-tests/utilities/user/exploration-editor.ts

```diff
@@ -212,7 +218,6 @@ const outcomeDestWhenStuckSelector =
 const intEditorField = '.e2e-test-editor-int';
 const setAsCheckpointButton = '.e2e-test-checkpoint-selection-checkbox';
 const tagsField = '.e2e-test-chip-list-tags';
-const uploadAudioButton = '.e2e-test-accessibility-translation-upload-audio';
```

Comment:
This was used in the old UI, now addManualVoiceoverButton selector will operate.